packages/styles/src/elements/paragraph.scss

```diff
@@ -0,0 +1,13 @@
+@use 'sass:map';
+@use '../functions/tokens';
+@use '../tokens/elements';
+
+tokens.$default-map: elements.$post-paragraph;
+
+p {
+  margin-block-start: tokens.get('paragraph-margin-block-start');
```

Comment:
We need to add a margin bottom here as well.

Reply:
It is added.